src/GraphQL.Tests/Serialization/SerializationTestBase.cs

```diff
@@ -0,0 +1,8 @@
+namespace GraphQL.Tests.Serialization
+{
+    public abstract class SerializationTestBase
```

Comment:
Remove?

Reply:
Do you think it's worth implementing any tests here? I'm not sure whether to bother or just wait for any potential issues to come up as per convo with @joemcbride earlier.

Reply:
There is simply no point in this blank stub.

Reply:
Yea it was a todo I had to write some tests. There's a few custom things happening in the serialization projects with custom converters, etc. Anyway, I've renamed this to `DocumentWriterTests` and done a super basic test and if needed we can add to that foundation in the future where we are lacking coverage.

Reply:
Good